vendor/ruby/astexport.rb

```diff
@@ -133,7 +133,9 @@ def visit(node)
     when :var_ref
       { ast_type: 'var_ref', ref: visit(node[1]) }
     when :arg_paren
-      { ast_type: 'arg_paren', args: visit(node[1]) }
+      type, args = node
+
+      { ast_type: type, args: (args.nil? ? nil : visit(args)) }
```

Comment:
As an alternative, the visitor could ignore `nil` nodes, but I'm not sure if that's a good solution.

Reply:
Having nil ternary here would make sense since we know that it can be nil. if we put it in visit function then it would add nil's at places where it is not clear to us that it can be nil or not.